README.md

```diff
@@ -150,6 +150,9 @@ A complete list of configuration options can be found below:
 | `proxyUri` | Optional URI of a proxy to use. Default: `null`, example: `http://proxy:81`.<br />*Since 2.0.0* |
 | `proxyUser` | Optional username for proxy authentication. Default: `null`.<br />*Since 2.0.0* |
 | `proxyPassword` | Optional password for proxy authentication. Default: `null`.<br />*Since 2.0.0* |
+| `cacheEnabled` | Enable caching of queries to keycloak to improve performance. Default: `false`.<br />*Since 2.2.0* |
+| `maxCacheSize` | Maximum size of the cache. Least used entries are evicted when this limit is reached. Default: `500`.<br />*Since 2.2.0* |
+| `cacheExpirationTimeout` | Time after which a cached entry is evicted. Default: `PT15M (15 minutes)`.<br />*Since 2.2.0* |
```

Comment:
What is the exact format of the cache expiration timeout? Any link to e.g. the corresponding ISO standard?

Reply:
It's the [default format of java.time.Duration.parse](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence-)

I'll add a few more examples and a link to the documentation.

Reply:
What do you think about configuring the timeout as simple integer / long value? And have it interpreted as minutes/ seconds, e.g. `cacheTimeoutMin` or `cacheTimeoutSec`. Then we won't need to check for a correct format and keep things as simple as possible from the outside. This could make configuration less error-prone. What do you think?

Reply:
Sounds reasonable. `cacheTimeoutMin` is indeed clearer and less confusing in this context. I'll make the change.

Reply:
Done